module/metrics.go

```diff
@@ -191,12 +191,18 @@ type VerificationMetrics interface {
 	// It increases the total number of result approvals.
 	OnResultApproval()
 
-	// LogVerifiableChunkSize is called whenever a verifiable chunk is shaped for a specific
-	// chunk. It adds the size of the verifiable chunk to the histogram. A verifiable chunk is assumed
-	// to capture all the resources needed to verify a chunk.
-	// The purpose of this function is to track the overall chunk resources size on disk.
-	// Todo wire this up to do monitoring (3183)
-	LogVerifiableChunkSize(size float64)
+	// OnFinalizedBlockReceived is called whenever a finalized block arrives at the assigner engine.
+	// It increments the total number of finalized blocks.
+	OnFinalizedBlockReceived()
+
+	// OnChunksAssigned is called whenever a chunks assigned to this verification node by applying chunk assignment on an
+	// execution result.
+	// It increases the total number of assigned chunks by the input.
+	OnChunksAssigned(chunks int)
+
+	// OnChunkProcessed is called whenever a chunk is pushed to the chunks queue by the assigner engine.
+	// It increments the total number of sent chunks.
+	OnChunkProcessed()
```

Comment:
Set of metrics methods added to `verification` metric collector to account for `assigner` engine.